app/code/community/Drip/Connect/Model/Transformer/Order.php

```diff
@@ -49,11 +49,15 @@ protected function getCommonOrderData()
             'occurred_at' => (string) Mage::helper('drip_connect')->formatDate($this->order->getUpdatedAt()),
             'items' => $this->getOrderItemsData(),
             'billing_address' => $this->getOrderBillingData(),
-            'shipping_address' => $this->getOrderShippingData(),
             'items_count' => floatval($this->order->getTotalQtyOrdered()),
             'magento_source' => (string) Mage::helper('drip_connect')->getArea(),
         );
 
+        $shipping = $this->getOrderShippingData();
+        if ($shipping !== null) {
+          $data['shipping_address'] = $shipping;
```

Comment:
Per https://github.com/DripEmail/magento-m2-extension/pull/51/files, do we also need to make `total_shipping` conditional?

Reply:
It looks to me like `total_shipping` comes through as `0` for orders for virtual products (that's what is happening tests). That should be fine, I think. The conditional here is so we just skip the `shipping_address` key altogether when there is no shipping.

Reply:
I took total_shipping out because, as Ian noticed, it comes through as zero, but I made the command decision to remove total_shipping since it made little sense to include the field since it's just noise.

I don't feel the divergence is bad, but if we need to be identical between the two, I'm more inclined to just hardcode the field to zero in M2, and only modify it if there's a possibility of actually calculating it.